packages/makeswift-next-plugin/index.js

```diff
@@ -81,7 +82,38 @@ module.exports =
       },
     }
 
-    return withTmInitializer(NEXT_TRANSPILE_MODULES_MODULES, {
-      resolveSymlinks,
-    })(enhancedConfig)
+    const nextVersion = require('next/package.json').version
+
+    if (satisfies(nextVersion, '<12.2.0')) {
+      throw new Error(
+        'Makeswift requires a minimum Next.js version 12.2.0. Please update to Next.js version ^12.2.0 if you want to use Next.js 12, or version ^13.0.0 if you want to use Next.js v13.',
```

Comment:
Some copy suggestions:

Makeswift requires a minimum Next.js version of 12.2.0. Please upgrade to Next.js version ^12.2.0 if you want to use Next.js v12, or version ^13.0.0 if you want to use Next.js v13.

Reply:
Updated